fixpatches/src/commonMain/kotlin/com/saveourtool/sarifutils/adapter/SarifFixAdapter.kt

```diff
@@ -215,42 +232,92 @@ class SarifFixAdapter(
      * @param fileReplacementsList list of replacements from all rules
      * @param targetFiles list of target files
      */
-    private fun applyReplacementsToFiles(fileReplacementsList: List<FileReplacements>, targetFiles: List<Path>): List<Path> = fileReplacementsList.mapNotNull { fileReplacements ->
-        val targetFile = targetFiles.find {
-            val fullPathOfFileFromSarif = if (!fileReplacements.filePath.adaptedIsAbsolute()) {
-                fs.canonicalize(sarifFile.parent!! / fileReplacements.filePath)
+    @Suppress(
+        "MaxLineLength",
+        "TOO_LONG_FUNCTION",
+    )
+    private fun applyReplacementsToFiles(
+        fileReplacementsList: List<FileReplacements>,
+        targetFiles: List<Path>,
+    ): List<Path> {
+        if (fileReplacementsList.isEmpty()) {
```

Comment:
I think, we have already such check in `process()` func: `if (runReplacements.isEmpty())`, so basically it's redundant
however we can let it, for generality of method

Reply:
Addressed with #48.